omx/gstomxaudiodec.c

```diff
@@ -1043,7 +1043,13 @@ gst_omx_audio_dec_handle_frame (GstAudioDecoder * decoder, GstBuffer * inbuf)
   GST_DEBUG_OBJECT (self, "Handling frame");
 
   if (self->downstream_flow_ret != GST_FLOW_OK) {
-    return self->downstream_flow_ret;
+    if (self->downstream_flow_ret == GST_FLOW_FLUSHING) {
```

Comment:
Maybe add a comment to tell why this is done

Reply:
Ready in commit cbfdccf4044eccd904cb6f41089223f313fa60a9